src/AutoRest.CSharp/LowLevel/Output/ConvenienceMethodOmittingMessage.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using AutoRest.CSharp.Common.Input;
+
+namespace AutoRest.CSharp.Output.Models
+{
+    internal record ConvenienceMethodOmittingMessage
```

Comment:
Seems this class only provide some static string message, is it possible to combine it with  ConvenienceMethodGenerationInfo class?

Reply:
The info is for internally passing information around when building those methods, this message is the final generated type, and there is an issue tracking the things we want to add into this message (wiki links, faqs, etc)